firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarState.swift

```diff
@@ -150,35 +150,14 @@ struct ToolbarState: ScreenState, Equatable {
         case SearchEngineSelectionActionType.didTapSearchEngine:
             return handleDidTapSearchEngine(state: state, action: action)
 
+        case SearchEngineSelectionMiddlewareActionType.didSelectAlternativeSearchEngine:
+            return handleDidSelectAlternativeSearchEngine(state: state, action: action)
+
         default:
             return state
         }
     }
 
-    private static func handleDidTapSearchEngine(state: Self, action: Action) -> ToolbarState {
```

Comment:
FYI this was just moved below for consistent organization (no changes) to the implementation.